### ISSUE TITLE
function-url-quotes does not work with space inside parentheses

### ISSUE THREAD

**Issue by in-in** [stylelint/stylelint#1177]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

`function-url-quotes` does not show warning if function have space inside parentheses

> Which rule, if any, is this issue related to?

`function-url-quotes`

> What CSS is needed to reproduce this issue?

```css
.foo {
  background-image: url( "bar.jpg" );
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "function-url-quotes": "none"
  }
}
```

> Which version of stylelint are you using?

linter-stylelint:  2.7.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Atom editor(1.7.3) and linter-stylelint plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

Warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

No warnings to be flagged.

![57c8d](https://cloud.githubusercontent.com/assets/8797432/14992363/75fa37a0-116e-11e6-9f25-55f1a30fc52e.jpg)

**Comment by ntwb** [comment]
Confirmed, I've added some failing tests in #1178

**Comment by ntwb** [comment]
Fixed in https://github.com/stylelint/stylelint/commit/fb4b8a00f7f38e5834f9e16665b31217c0797b87
